plugins/inputs/windows_event_log/wineventlog/utils_test.go

```diff
@@ -15,51 +15,54 @@ import (
 
 func TestPayload_Value(t *testing.T) {
 	// one null test
+	NumbersOfBytesPerCharacter = 0
```

Comment:
I'm fine with this because I understand that we need to reset the state for the tests, but just to make that more obvious why we're setting this, can we pull this out into a helper function like
```go
func resetState() {
    NumberOfBytesPerCharacter = 0
}
```
That makes it very obvious why we set this, and then if/when more tests get added here it's easier to remember to also do this in new tests.